src/components/SidebarTabs/SharingSidebarTab.vue

```diff
@@ -60,6 +60,12 @@
 						</template>
 						{{ t('forms', 'Copy to clipboard') }}
 					</NcActionLink>
+					<NcActionButton @click="openQrDialog(getPublicShareLink(share))">
```

Comment:
```suggestion
					<NcActionButton @click="openQrDialog(share)">
```

Better to handle the logic in the method than in the template.